data/mods/Tanks/items.json

```diff
@@ -1,27 +1,41 @@
 [
   {
+    "abstract": "artillery_casing",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "A large artillery casing.",
+    "stackable": true,
+    "material": "steel",
```

Comment:
Are they not brass? Hardly critical here. You could also consider `copy-from` the `abstract` casing